docs/deployment_practices.md

```diff
@@ -91,20 +101,36 @@ Here's an Apache example that would exclude the landing page from logging:
+
+It's preferable for the landing page to have its own segmented environment instead of hosting it alongside other sites running potentially vulnerable software or content management systems. Check that user and group file permissions are locked down and that modules or gateway interfaces for dynamic scripting languages are not enabled. You don't want any unnecessary code or services running as this increases the attack surface.
+
 ### Minimum requirements for the SecureDrop environment
 
-*   The Application and Monitor servers should be dedicated physical machines, not virtual machines.
-*   A trusted location to host the servers. The servers should be hosted in a location that is owned or occupied by the organization to ensure that their legal can not be bypassed with gag orders.
-*   The SecureDrop servers should be on a separate internet connection or completely segmented from corporate network.
-*   All traffic from the corporate network should be blocked at the SecureDrop's point of demarcation.
-*   Video monitoring should be recorded of the server area and the organizations safe.
-*   Journalist should ensure that while using the air-gapped viewing station they are in an area without video cameras.
-*   An established monitoring plan and incident response plan. Who will receive the OSSEC alerts and what their response plan will be? These should cover technical outages and a compromised environment plan.
+* The Application and Monitor servers should be dedicated physical machines, not virtual machines.
+* A trusted location to host the servers. The servers should be hosted in a location that is owned or occupied by the organization to ensure that their legal can not be bypassed with gag orders.
+* The SecureDrop servers should be on a separate internet connection or completely segmented from corporate network.
+* All traffic from the corporate network should be blocked at the SecureDrop's point of demarcation.
+* Video monitoring should be recorded of the server area and the organizations safe.
+* Journalist should ensure that while using the air-gapped viewing station they are in an area without video cameras.
+* An established monitoring plan and incident response plan. Who will receive the OSSEC alerts and what their response plan will be? These should cover technical outages and a compromised environment plan.
 
 ### Suggested
 
-*   For publicly advertised SecureDrop instances display the Source Interface's hidden service onion address on all of the organization public pages.
-*   Mirror the Tor Browser and Tails so sources do not have to visit [torproject.org](https://www.torproject.org) to download it.
+* For publicly advertised SecureDrop instances display the Source Interface's hidden service onion address on all of the organization public pages.
+* Mirror the Tor Browser and Tails so sources do not have to visit [torproject.org](https://www.torproject.org) to download it.
```

Comment:
:+1: Thanks for fixing the other locations. Not sure how that happened in the first place - probably someone's editor was misconfigured.